general/getDateTime.md

```diff
@@ -29,6 +29,8 @@ none
 
 ### Examples
 
+<a class="dlbtn" href="https://raw.githubusercontent.com/opentx/lua-reference-guide/master/general\getDateTime-example.lua">general\getDateTime-example</a>
```

Comment:
The `\` character in path needs to be converted to `/` character. My system (linux) already generates proper `/` char here.